torch/csrc/jit/codegen/cuda/scheduler/vectorize_helper.cpp

```diff
@@ -881,11 +944,42 @@ void ContiguousInnerDimensionsMapper::propagateSibling(
         from->toString());
     auto pos = std::distance(from->getRFactorDomain().begin(), from_it);
     sibling_rfactor_ids.push_back(to->getRFactorDomain()[pos]);
-    projected_extent_[to->getRFactorDomain()[pos]] =
-        getMappedExtent(from->getRFactorDomain()[pos]);
+    if (recording_) {
+      addProjectedExtent(
+          to->getRFactorDomain()[pos],
+          getMappedExtent(from->getRFactorDomain()[pos]));
+    }
   }
 
-  projected_rfactor_ids_[to] = sibling_rfactor_ids;
+  return MappedDomain::build(sibling_root_ids, sibling_rfactor_ids);
+}
+
+// MaxInfoSpanningTree functions
+void ContiguousInnerDimensionsMapper::propagateC2P(
+    TensorView* from,
+    TensorView* to) {
+  recording_ = true;
+  auto from_info = tv_infos_.at(from);
+  auto to_info = computeInfoC2P(from, to, from_info);
+  tv_infos_[to] = to_info;
```

Comment:
Do you need to set `recording_` back to false when you are done with `computeInfoC2P`?

Reply:
Is there a chance the max spanning tree would be recomputed? I thought max spanning tree is computed once only calling `computeInfo...` calls, then once that's done we'll never be computing the spanning tree again so we can just leave recording on.